index.html

```diff
@@ -8,6 +8,13 @@
         content="ZanyMen is an up-and-coming name in the market of for-rental wacky arm-waving inflatable men.">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
+    <!-- GOOGLE FONTS -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+        rel="stylesheet">
+
```

Comment:
Added 'Poppins' google font. Its sans sarif and looks a bit fancier.

Reply:
nice one, looks good!